spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestPositionDeletesTable.java

```diff
@@ -123,15 +126,9 @@ public static Object[][] parameters() {
     };
   }
 
-  public TestPositionDeletesTable(
-      String catalogName, String implementation, Map<String, String> config, FileFormat format) {
-    super(catalogName, implementation, config);
-    this.format = format;
-  }
-
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
+  @TempDir private Path temp;
```

Comment:
is this needed? I think you should be able to use the one defined in the super class

Reply:
updated